### DISCUSSION TITLE
status indicator

### DISCUSSION THREAD

**Discussion by ksdavidc** [remotely-save/remotely-save#181] [Ideas]
This has been working well for me, so thank you. Friendly suggestion:

It would be nice if there were some kind of status indicator. When I run the sync I usually get a few (4) notification banners for the first steps, but then it goes blank for a while, and then as the other steps come through the other notifications come up, but go away quickly, so I often don't notice or know if the sync has finished or not. 

It would be nice if there were some visible indication that the sync was in progress or had finished that was more persistent. (I imagine something like the icon changing to indicate the step number, and then become a certain color when the sync is finished, which would disappear after hovering or file change or something, but obviously there are probably better or easier ways to do it). I know that if I try to rerun and a sync is in progress there is a status banner, but this would initiate a new sync if it were already finished.

**Comment by fyears** [comment]
latest version has status bar
